configcatclient/configfetcher.py

```diff
@@ -1,14 +1,18 @@
 import requests
 import logging
 import sys
+
+from .datagovernance import DataGovernance
 from .version import CONFIGCATCLIENT_VERSION
+from .constants import *
 
 if sys.version_info < (2, 7, 9):
     requests.packages.urllib3.disable_warnings()
 
-BASE_URL = 'https://cdn.configcat.com'
+BASE_URL_GLOBAL = 'https://cdn-global.configcat.com'
+BASE_URL_EUONLY = 'https://cdn-eu.configcat.com'
 BASE_PATH = 'configuration-files/'
-BASE_EXTENSION = '/config_v4.json'
+BASE_EXTENSION = '/config_v5.json'
```

Comment:
Please avoid 'config_v5' string duplication.
Use the CACHE_KEY = 'config_v5' here or create a new constant for this string

Reply:
DONE